Sources/PowerSync/Protocol/db/CrudTransaction.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import PowerSyncKotlin
```

Comment:
The implementation here looks great to me. 
While this is not strictly required. I like to keep `PowerSyncKotlin` out of the files which define the Swift Protocols. Previously mixing these has made it very easy to leak PowerSyncKotlin into our public APIs - we aren't leaking anything here, but to make it a little easier to prevent in future it would be nice if we could perhaps declare `CrudTransactions` as a Protocol or typealias here and link it to the Kotlin implementation in a file in the `Kotlin` folder.